driver.sh

```diff
@@ -143,6 +143,12 @@ setup_variables() {
             OBJDUMP=${CROSS_COMPILE}objdump
             ;;
 
+        "x86")
+            config=defconfig
+            make_target=bzImage
+            export ARCH=i386
```

Comment:
rather than build `ARCH=i386 defconfig`, why don't we build `i386_defconfig`? Is there a difference? (Maybe doesn't matter)

Reply:
The configs are identical, but the output is smaller by 1 line.

Reply:
Is this export still needed then?

Reply:
I’d rather keep the export in case using a different config. `oldconfig` target with `ARCH` set to `i386` should produce correct configs for x86.